releasenotes/notes/botocore-46f5bf04f64777b4.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
```

Comment:
this should be an `upgrade`-level note mentioning the change and the backwards compatibility option as users using older python/node lambda client versions will have a breaking change